checks.d/zk.py

```diff
@@ -38,6 +39,15 @@ class ZKConnectionFailure(Exception):
     pass
 
 
+class ZKMetric(namedtuple('ZKMetric', ['name', 'value', 'm_type'])):
```

Comment:
This is a bit weird. You are using named tuples but you are not using the fact that attributes are named. You should just use regular tuples IMO.